docs/source-fabric/fundamentals/precision.rst

```diff
@@ -29,25 +29,34 @@ This is how you select the precision in Fabric:
     # This is the default
     fabric = Fabric(precision="32-true")
 
-    # Also FP32
+    # Also FP32 (legacy)
     fabric = Fabric(precision=32)
 
-    # FP32 as well
+    # FP32 as well (legacy)
     fabric = Fabric(precision="32")
 
-    # FP16 mixed precision
+    # Float16 mixed precision
     fabric = Fabric(precision="16-mixed")
 
-    # BFloat16 precision (Volta GPUs and later)
+    # Float16 half precision
+    fabric = Fabric(precision="16-true")
+
+    # BFloat16 mixed precision (Volta GPUs and later)
     fabric = Fabric(precision="bf16-mixed")
 
+    # BFloat16 true half precision (Volta GPUs and later)
+    fabric = Fabric(precision="bf16-true")
+
+    # 8-bit mixed precision (Hopper GPUs and later)
+    fabric = Fabric(precision="8-mixed")
```

Comment:
How confident are we with this naming choice? The library is overfitting to transformer models, so I think we shouldn't expose the name as a general choice for mixed precision training. I would suggest "8-mixed-te" or "8-mixed-transformer" to keep it short but specific. 
(just opening the discussion but not blocking)